### DISCUSSION TITLE
how to get a fire of button on melee

### DISCUSSION THREAD

**Discussion by cry-1337** [SmartlyDressedGames/Unturned-3.x-Community#4175] [Q&A]
rocketmod framework.

**Comment by GazziFX** [comment]
Do you want to handle every key press or only stabs?

**Reply by cry-1337** [reply]
only stabs

**Reply by DanielWillett** [reply]
Looks like you would have to do some patching.

If you only care that they used it but not if it hit something, patch UseableMelee.swing(). If you want to know if it hit something, you'll have to learn transpiling.
